.github/workflows/golang.yml

```diff
@@ -31,3 +37,44 @@ jobs:
           context: ./golang
           file: ./golang/Dockerfile
           platforms: ${{ steps.buildx.outputs.platforms }}
+
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+      -
+        name: Prepare
+        id: prep
+        run: |
+          DOCKER_IMAGE=tonistiigi/xx
+          VERSION=golang-edge
+          if [[ $GITHUB_REF == refs/tags/* ]]; then
+            VERSION=${GITHUB_REF#refs/tags/v}
+          fi
+          TAGS="${DOCKER_IMAGE}:golang-${VERSION}"
+          if [[ $VERSION =~ ^[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}$ ]]; then

```

Comment:
assuming tag is `golang/v*` we can just replace `/` with `-` and add constant `tonistiigi/xx:golang` as the latest version.

I prefer `golang-master` instead `golang-edge`. `golang-<branch>`

Reply:
Done and updated [my comment](https://github.com/tonistiigi/xx/pull/11#issue-486055664) to reflect that.